### PR TITLE
[PT] Improve Error Handling

### PR DESCRIPTION
**Pull request by dukex**
Fixing some Portuguese sentences

### CODE REVIEW

File: pt/lessons/advanced/error-handling.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ end
 
 ## Novos Erros
 
-Enquanto Elixir inclui uma série de tipos de erro nativos como `RuntimeError`, mantemos a capacidade de criar a nossa própria se precisamos de algo específico. Criar um novo erro é fácil com o `defexception/1` macro que aceita convenientemente `:message` opções para definir uma mensagem de erro padrão:
+Enquanto o Elixir inclui uma série de tipos de erro nativos, como `RuntimeError`, nós mantemos a capacidade de criar a nosso próprio se precisamos de algo específico. Criar um novo erro é fácil com o `defexception/1` macro que aceita convenientemente a opção `:message` para definir uma mensagem de erro padrão:
```
Comment (anderkonzen) [inline pt/lessons/advanced/error-handling.md]: "... capacidade de criar **o** nosso próprio se...", just a minor typo with the article used here.

File: pt/lessons/advanced/error-handling.md [hunk 2]
```diff
@@ -103,9 +103,9 @@ iex> try do
 
 ## Lançar
 
-Outro mecanismo para trabalhar com erros no Elixir é `throw` e `catch`. Na prática, isso não ocorre frequentemente no código mais recente de Elixir. Mas, é importante conhecer e entendê-los mesmo assim.
+Outro mecanismo para trabalhar com erros no Elixir é o `throw` e o `catch`. Na prática, isso não ocorre frequentemente no código mais recente de Elixir. Mas, é importante conhecer e entendê-los mesmo assim.
 
-A função `throw/1` nos dá a capacidade para sair de execução com um valor específico que podemos usar com o `catch`:
+A função `throw/1` nos dá a capacidade para sair da execução com um valor específico que podemos `catch`(pegar) e usar:
```
Comment (anderkonzen) [inline pt/lessons/advanced/error-handling.md]: Another minor typo: we could add a space between `catch` and "(pegar)" words.

File: pt/lessons/advanced/error-handling.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-version: 0.9.0
+version: 1.0.0
 title: Tratamento de Erros
 ---
 
-Embora seja mais comum o retorno da tupla `{: erro, reason}`, Elixir suporta exceções e nesta lição veremos como lidar com erros e os diferentes mecanismos disponíveis para nós.
+Embora seja mais comum o retorno da tupla `{:error, reason}`, o Elixir suporta exceções e nesta lição veremos como lidar com erros e os diferentes mecanismos disponíveis para nós.
 
-Em geral, a convenção em Elixir é criar uma função (`example/1`) que retorna` {: ok, result} `e` {: error, reason} `e uma função separada (`example!/1`) que retorna o `result` desembrulhado ou gerará um erro.
+Em geral, a convenção em Elixir é criar uma função (`example/1`) que retorna` {:ok, result} ` e ` {:error, reason} ` e uma função separada (`example!/1`) que retorna o `result` desempacotado ou levanta um erro.
```
Comment (anderkonzen) [inline pt/lessons/advanced/error-handling.md]: I'm being a bit picky picky here 😄 but we could also remove spaces between the apostrophe and curly brackets ` {:ok, result} ` and here ` {:error, reason} `.

**Comment by dukex** [comment]
Thanks @anderkonzen, nice touch! I apply your suggestions

**Pull request merged**